test/getCommands.spec.js

```diff
@@ -0,0 +1,50 @@
+const path = require('path');
+const Module = require('module');
+const expect = require('chai').expect;
+const getCommands = require('../src/getCommands');
+const findPlugins = require('../src/findPlugins');
+const mockRequire = require('mock-require');
+const mockFs = require('mock-fs');
+
+const commands = getCommands();
+const filesPath = path.join(__dirname, 'fixtures', 'files');
+const testPlugin = findPlugins(filesPath);
+const testPluginPath = path.join(__dirname, '..', 'node_modules', 'rnpm-plugin-test');
+const singlePlugin = require(path.join(filesPath, 'plugin'));
+const multiPlugin = require(path.join(filesPath, 'plugins'));
+
+describe('getCommands', () => {
+
+  before(() => {
+    mockRequire(path.join(__dirname, '..'), {});
+    mockRequire(
+      path.join(__dirname, '..', 'package.json'),
+      require(path.join(filesPath, 'package'))
+    );
```

Comment:
That comment was invalid anyway since I didn't realise mock-fs does not work well with mock-fs.